artiq/coredevice/phaser.py

```diff
@@ -549,6 +568,50 @@ def get_dac_temperature(self) -> TInt32:
         """
         return self.dac_read(0x06, div=257) >> 8
 
+    @kernel
+    def dac_sync(self):
+        """Trigger DAC synchronisation for both output channels.
```

Comment:
Explain that this only synchronizes the DAC-internal functionality where `sif_sync` is enabled enabled.

Reply:
See below. It is probably worth adding another paragraph here.